src/coreComponents/linearAlgebra/utilities/LinearSolverParameters.hpp

```diff
@@ -186,21 +186,27 @@ struct LinearSolverParameters
       rigidBodyModes, ///< Rigid body modes
     };
 
-    integer maxLevels = 20;                         ///< Maximum number of coarsening levels
-    CycleType cycleType = CycleType::V;             ///< AMG cycle type
-    SmootherType smootherType = SmootherType::fgs;  ///< Smoother type
-    CoarseType coarseType = CoarseType::direct;     ///< Coarse-level solver/smoother
-    string coarseningType = "HMIS";                 ///< Coarsening algorithm
-    integer interpolationType = 6;                  ///< Coarsening algorithm
-    integer numSweeps = 2;                          ///< Number of smoother sweeps
-    integer numFunctions = 1;                       ///< Number of amg functions
-    integer aggresiveNumLevels = 0;                 ///< Number of levels for aggressive coarsening.
-    PreOrPost preOrPostSmoothing = PreOrPost::both; ///< Pre and/or post smoothing
-    real64 threshold = 0.0;                         ///< Threshold for "strong connections" (for classical and smoothed-aggregation AMG)
-    integer separateComponents = false;             ///< Apply a separate component filter before AMG construction
+#if defined(GEOSX_USE_HYPRE_CUDA) || defined(GEOSX_USE_HYPRE_HIP)
+    string coarseningType = "PMIS";                             ///< Coarsening algorithm
+    SmootherType smootherType = SmootherType::l1jacobi;         ///< Smoother type
+#else
+    string coarseningType = "HMIS";                             ///< Coarsening algorithm
+    SmootherType smootherType = SmootherType::l1sgs;            ///< Smoother type
```

Comment:
This options make sense if GEOS uses hypre as linear algebra interface. What happens if Trilinos or PETSc are chosen? I assume such options would have no effect with Trilinos and PETSc relying on the respective default parameters.

Reply:
Good point, I didn't check that. Maybe @klevzoff can confirm?

Reply:
PETSc currently does not use any AMG parameters from input. Trilinos will use regular SGS/Jacobi instead of l1.